index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Green Earth Summit</title>
+</head>
+<body>
+    <navbar>
+        <div class="upper-nav">
+            <ul class="top">
+                <li class="fb"><img  id="fb" src="assets/fb (1).png" alt="facebook icon"/></li>
+                <li class="twitter"><img id="twitter" src="assets/twitter.png" alt="twitter icon"></li>
+                <li class="pa"><p>English</p></li>
+                <li class="pa"><p>MyPage</p></li>
+                <li class="pa"><p>Logout</p></li>
+            </ul>
+        </div>
+        <div class="hamburger">
+            <ul>
+            <li class="bar"></li>
+            <li class="bar"></li>
+            <li class="bar"></li>
+        </ul>
+        </div>
+        <div  class="flex" >
+            <a  class="logo" href="index.html"><img  id="logo"  src="assets/Earth.png"/></a>
+            <ul class="nav-menu">
+              <li class="nav-item">
+              <a class="nav-link" id="nav1"  href="about.html">About</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">Program</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="#">Join</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Sponsor</a>
+              </li>
+            <li class="nav-item">
+              <a class="nav-link"  href="#">News</a>
+            </li>         
+            <li class="nav-item GE">
+                <a class="nav-link" id="ge" href="#">Campaign</a>
+              </li>    
+          </ul>
+          </div>
+    </navbar>
+   <!--Header-->
+   <header>
+    <h1 class="heading1">"Hello!Sharing world"</h1>
+    <h1 class="heading2">Green Earth Global Summit 2023</h1>
```

Comment:
- [ ] As requested by the previous reviewer, this has not been implemented. You should always have one `h1` tag per page as this is the main heading.

- [OPTIONAL]  instead of having it as te previous reviewer requested like 
```
    <h2 class="heading1about">"Hello!Sharing world"</h2>
    <h1 class="heading2">Green Earth Global Summit 2023</h1>
```
I suggest you have it like 

```
    <p class="heading1about">"Hello!Sharing world"</p>
    <h1 class="heading2">Green Earth Global Summit 2023</h1>
```
This is because, all headings on a page should be hierarchical. This means you should not have an `h1` below and `h2` but the reverse is true. 👍🏾